cmd/fs-v1.go

```diff
@@ -1019,8 +1032,14 @@ func (fs *FSObjects) ListObjects(ctx context.Context, bucket, prefix, marker, de
 			// object string does not end with "/".
 			return !hasSuffix(object, slashSeparator)
 		}
+		isLeafDir := func(bucket, object string) bool {
```

Comment:
Provide an explanation of what this does?

Reply:
> Provide an explanation of what this does?

Done